Tasks/VsTest/typings/vsts-task-lib.d.ts

```diff
@@ -1,12 +1,12 @@
 declare module 'vsts-task-lib/taskcommand' {
 	export class TaskCommand {
-	    constructor(command: any, properties: any, message: any);
-	    command: string;
-	    message: string;
-	    properties: {
-	        [key: string]: string;
-	    };
-	    toString(): string;
+		constructor(command: any, properties: any, message: any);
```

Comment:
i dont think the d.ts file needs to be checked in. intellisense will come after npm install.